node-notes-cli/app.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const dataJson = require('./data.json');
+
+const operation = process.argv[2];
+const choice = process.argv[3];
+
+const jsonNotes = dataJson.notes;
+const notesIndex = Object.keys(jsonNotes);
+
+if (operation === 'read') {
+  fs.readFile('data.json', 'utf8', (err, data) => {
+
+    if (err) throw err;
```

Comment:
If you `require()` the data like you did on line 4, you can skip reading the file.